src/util.js

```diff
@@ -6,18 +6,20 @@ export function submitForm(form, { headers, cors, strict } = {}) {
 	let method = form.getAttribute("method");
 	method = method ? method.toUpperCase() : "GET";
 	let uri = form.getAttribute("action");
-	let payload = serializeForm(form);
+	let body;
 
 	if(method === "GET") {
 		if(uri.indexOf("?") !== -1) {
 			throw new Error("query strings are invalid within `GET` forms' " +
 					"`action` URI; please use hidden fields instead");
 		}
-		uri = [uri, payload].join("?");
+		uri = [uri, serializeForm(form, new StringParams())].join("?");
+	} else if(form.getAttribute("enctype") === "multipart/form-data") {
+		body = serializeForm(form, new FormDataParams());
```

Comment:
I need to test this out, but theoretically we should be able to use `body = new FormData(form)` here. Then it is not necessary to call the `serializeForm` function